client/lib/signup/step-actions/index.js

```diff
@@ -929,6 +929,14 @@ export function isDomainFulfilled( stepName, defaultDependencies, nextProps ) {
 	}
 }
 
+export function excludePlansStep( submitSignupStep ) {
+	submitSignupStep(
+		{ stepName: 'plans', cartItem: undefined, wasSkipped: true },
+		{ cartItem: undefined }
+	);
+	flows.excludeStep( 'plans' );
```

Comment:
Not really sure but maybe we want to log somewhere the fact that the step has been skipped? Or maybe not, technically is always skipped 😅